ihatemoney/forms.py

```diff
@@ -181,6 +199,10 @@ def save(self, bill, project):
         bill.external_link = self.external_link.data
         bill.date = self.date.data
         bill.owers = [Person.query.get(ower, project) for ower in self.payed_for.data]
+        bill.original_currency = self.original_currency.data
+        bill.original_amount = self.currency_helper.exchange_currency(
```

Comment:
This seems weird to me: it's named `original_amount` but it actually stored the converted amount, right?

Reply:
If so, we might want to rephrase how this is named. What about `converted_amount`?